lib/ui/views/explore/explore_page_view.dart

```diff
@@ -134,7 +111,8 @@ class _ExploreTabsState extends State<ExploreTabs>
   @override
   void initState() {
     super.initState();
-    _tabController = TabController(length: exploreTabs.length, vsync: this);
+    _tabController =
+        TabController(length: ExploreTabKey.values.length, vsync: this);
```

Comment:
Trailing comma

Reply:
Personally I think this syntax is fine. If it was just the parameters on the line below I would agree but its the whole statement so seems fine